app/views/super_admin/content.py

```diff
@@ -71,15 +72,20 @@ def placeholder_upload():
             wpercent = (basewidth / float(img.size[0]))
             hsize = int((float(img.size[1]) * float(wpercent)))
             img = img.resize((basewidth, hsize), PIL.Image.ANTIALIAS)
-            os.mkdir(app.config['TEMP_UPLOADS_FOLDER'])
+            try:
+                os.mkdir(app.config['TEMP_UPLOADS_FOLDER'])
```

Comment:
@SaptakS should i remove the try..catch block if you want, it makes no difference anyways.